### PR TITLE
Add pytximport

### PR DESCRIPTION
**Pull request by maltekuehl**
`pytximport` is a tool to import transcript quantification files from RNA sequencing, correct them for differential transcript usage bias and sum expression at the gene level. It is a Python implementation of `tximport`.
`pytximport` is hosted here: [https://github.com/complextissue/pytximport/](https://github.com/complextissue/pytximport/)

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by maltekuehl** [comment]
@BiocondaBot please add label

File: recipes/pytximport/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - pytximport --help
+
+about:
+  home: https://pytximport.readthedocs.io/en/latest/start.html
+  dev_url: https://github.com/complextissue/pytximport
+  doc_url: https://pytximport.readthedocs.io
+  summary: pytximport - gene count estimation from alignment-free quantification
+  description: |
+    pytximport is a Python port of tximport that allows users to import transcript counts from tools such as kallisto and Salmon, correct them for differential isoform usage, and summarize them at the gene level.
+  license: GPL-3
```
Comment (martin-g) [inline recipes/pytximport/meta.yaml:52]:
```suggestion
  license: GPL-3.0-only
```

File: recipes/pytximport/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  license_family: GPL
+  license_file: LICENSE
+
+extra:
+  additional-platforms:
+    - linux-aarch64
```
Comment (martin-g) [inline recipes/pytximport/meta.yaml:58]: The recipe is noarch. There is no need of additional platforms.

**Comment by maltekuehl** [comment]
I've integrated all requested changes and re-requested the review. If anything else needs to change, please let me know & I will be happy to update the PR.

File: recipes/pytximport/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytximport-{{ version }}.tar.gz
+  sha256: 7a32d5cf5cd9582fe832661b81f0159896d69b7d52dd66ca55d74f1ec48f1606
+
+build:
+  noarch: generic
```
Comment (mencian) [inline recipes/pytximport/meta.yaml:13]:
```suggestion
  noarch: python
```

**Review by mencian — APPROVED** [review]
LGTM, thanks @maltekuehl @martin-g!

**Pull request merged**